gui/src/app/mod.rs

```diff
@@ -300,6 +300,12 @@ impl App {
                 )
             }
             Message::View(view::Message::Menu(menu)) => self.set_current_panel(menu),
+            Message::View(view::Message::OpenUrl(url)) => {
+                if let Err(e) = open::that(&url) {
```

Comment:
halloy is using that_detached https://docs.rs/open/latest/open/fn.that_detached.html
It may be better to use it instead.

Reply:
Yep that seems better thanks, will change it. Would be good to test it also on Windows.